Brewfile

```diff
@@ -1,2 +1,2 @@
-brew "node@10"
+brew "node@12"
```

Comment:
Is this a change in recommended the node version? Should we also be updating documentation for setting up machines?